src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -105,26 +105,45 @@ def get(self, request, organization):
         :auth: required
         """
         stats_period = request.GET.get("groupStatsPeriod")
+        try:
+            start, end = get_date_range_from_params(request.GET)
+        except InvalidParams as e:
+            return Response({"detail": six.text_type(e)}, status=400)
```

Comment:
You could use `rest_framework.exceptions.ParseError` here it would be a bit terser as `raise ParseError(detail=six.text_type(e))`

Reply:
will do